EasyPost/Models/API/Batch.cs

```diff
@@ -47,6 +47,7 @@ internal Batch()
         [CrudOperations.Update]
         public async Task<Batch> AddShipments(Dictionary<string, object> parameters)
         {
+            // parameters = parameters.Wrap("batch");  // TODO: Update docs to remove wrapped "batch" key
```

Comment:
`batch` should be wrapped and included here

Reply:
It does not work when wrapped.
<img width="1155" alt="image" src="https://user-images.githubusercontent.com/17054780/197587559-1060ff2f-0261-445f-88a0-ba81cf206666.png">

Reply:
Hmm... I was able to successfully do this in Postman before posting this.

We may need to do some additional testing to find the actual answer.